modules/administration-guide/partials/proc_restoring-a-che-instance-from-a-backup-using-the-checlusterrestore-object.adoc

```diff
@@ -0,0 +1,66 @@
+[id="restoring-a-{prod-id-short}-instance-from-a-backup-using-the-checlusterrestore-object_{context}"]
+= Restoring a {prod-short} instance from a backup using the CheClusterRestore custom object
+
+You can restore a {prod-short} instance from a backup snapshot using custom resources. A new `CheClusterRestore` custom object is required every time you restore a {prod-short} instance from a backup snapshot; that is, editing an already consumed `CheClusterRestore` custom object, during or after backing up, has no effect.
+
+include::snip_warning-backup-snapshots-are-bound-to-specific-cluster.adoc[]
+
+NOTE: The `CheClusterRestore` custom object is unusable for recovering a {prod-short} instance of a different version of {prod-short}. To recover a {prod-short} instance of an earlier version of {prod-short} by using `{prod-cli}`, see xref:restoring-a-backed-up-instance-of-an-earlier-version-of-{prod-id-short}_{context}[].
+
+.Prerequisites
+
+* Backup snapshot of a {prod-short} instance on any of the following:
+** xref:backups-of-che-instances-to-an-sftp-backup-server.adoc[External SFTP backup server]
+** xref:backups-of-che-instances-to-amazon-s3.adoc[Amazon S3]
+** xref:backups-of-che-instances-to-a-rest-backup-server.adoc[External REST backup server]
+** xref:backups-of-che-instances-to-the-internal-backup-server.adoc[Internal backup server]
+
+.Procedure
+
+. Create the `CheClusterRestore` custom object, which restores a backup:
++
+[source,yaml,subs="+attributes,+quotes"]
+----
+apiVersion: org.eclipse.che/v1
+kind: CheClusterRestore
+metadata:
+  name: {prod-id}-restore
+  namespace: {prod-namespace}
+spec:
+  backupServerConfigRef: backup-server-configuration <1>
+  snapshotId: _<snapshot_id>_ <2>
+----
+<1> Name of the `CheBackupServerConfiguration` object that defines what backup server to use.
+<2> Optional parameter defining the Snapshot ID to restore from. The default value is the last snapshot in the backup repository on the backup server.
++
+[TIP]
+====
+If you intend to reuse a `name` for `CheClusterRestore` custom objects, first make sure to delete any existing custom object with the same `name`. To delete it on the command line, use {orch-cli}:
+[source,shell,subs="+attributes,+quotes"]
+----
+$ {orch-cli} delete CheClusterRestore _<name>_ -n {prod-namespace}
+----
+====
+
+. Wait until the recovery process ends.
++
+TIP: If errors occur in your browser after the recovery, clean up the browser data for the {prod-short} domain.
```

Comment:
I'd move it to step 4: `Check that the {prod-short} instance has been recovered`